src/hugr/views/sibling_subgraph.rs

```diff
@@ -762,7 +763,12 @@ mod tests {
 
     fn build_3not_hugr() -> Result<(Hugr, Node), BuildError> {
         let mut mod_builder = ModuleBuilder::new();
-        let func = mod_builder.declare("test", FunctionType::new_endo(type_row![BOOL_T]).into())?;
+        let func = mod_builder.declare(
+            "test",
+            FunctionType::new_endo(type_row![BOOL_T])
```

Comment:
That this delta wasn't needed immediately following #734 is a kind of "bug" fixed by this PR